### PR TITLE
auto label 'azure-spring' for PRs with github labeler action

### PR DESCRIPTION
**Pull request by stliu**
This pr uses [Github Labeler action](https://github.com/marketplace/actions/labeler) to add labels for PRs

https://github.com/Azure/azure-sdk-for-java/issues/28019

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Add 'azure-spring' label to any change to sdk/spring folder
+azure-spring:
+  - sdk/spring/**/*
+  - sdk/bomx/spring-cloud-azure-dependencies/**/*
```
Comment (saragluna) [inline .github/labeler.yml:4]: bom

**Review by saragluna — CHANGES_REQUESTED** [review]
IIRC the PR changes codes related to sdk/spring have already added the azure-spring label automatically. https://github.com/Azure/azure-sdk-for-java/pulls?q=is%3Apr+is%3Aopen+label%3Aazure-spring

File: .github/labeler.yml (hunk 1 above)
Comment (stliu) [reply .github/labeler.yml]: fixed

**Review by JimSuplizio — COMMENTED** [review]
@stliu I would like to know what purpose this serves and why this is necessary? In the future, I would appreciate that being in the description of PRs of this nature.

**Review by weshaggard — CHANGES_REQUESTED** [review]
At this time we are not ready to start taking a dependency on github actions/workflows. 

For tagging we already use the fabric bot which already handles tagging of PRs and issues based on paths. See https://github.com/Azure/azure-sdk-for-java/blob/main/.github/fabricbot.json if you need help please talk with @jsquire.

**Comment by stliu** [comment]
okay, seems this already done by another bot, closing this one

**Comment by jsquire** [comment]
The main challenge with Fabric Bot to replicate this is the lack of support for glob or regex patterns for path identification.    It supports only `StartsWith` semantics, which means that each child path intended to be covered by an individual Fabric Bot entry in the rule.

**Pull request closed without merging**